packages/mobile-top-nav/src/assets/img/ia-icon.js

```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from 'lit-element';
+import './icons';
+
+class IAIcon extends LitElement {
+  constructor() {
+    super();
+    this.icon = '';
+    this.fill = 'fff';
+  }
+
+  static get properties() {
+    return {
+      icon: { type: String },
+      fill: { type: String },
+    };
+  }
+
+  get icons() {
+    return {
+      audio: html`<icon-audio .fill=${this.fill}></icon>`,
```

Comment:
Instead of importing all of the icons as individual tags, you can just import the source and render it directly out:
```
audio: html`<svg ....></svg>`
...
```

Reply:
I can only imagine the file with bulky wrap around lines of  `<svg ....></svg>` 😵

Reply:
This was just an example. You can also just import the svgs from a different file. My point is we don't need individual components for every icon.
```
import audio_icon from './audio.svg';
...
audio: html`${audio_icon}`
```

Reply:
You cannot import an SVG file due to MIME type mismatches. The new static implementation I've created is as close to plain SVG as we can go without relying on another library.

Reply:
Ah you do this instead:

```
// audioImageFile.js
import { html } from 'lit-html';

const image = html`
  <svg></svg>
`;

export default image;
---
// consumer.js
import audioImage from './audioImageFile.js';

audio: audioImage
```

Reply:
Yeah, that's what I'm doing now. I tagged you on that thread to have you review the new implementation. They're all pulled together via [src/assets/img/static_icons.js](https://github.com/internetarchive/iaux/blob/feature-mobile-topnav/packages/mobile-top-nav/src/assets/img/static_icons.js)